packages/syft/src/syft/core/tensor/passthrough.py

```diff
@@ -55,13 +46,65 @@ def __len__(self) -> int:
         return len(self.child)
 
     @property
-    def shape(self):
-        return tuple(self.child.shape)
+    def shape(self) -> Union[TypeTuple[Any, ...], List[Any]]:
+        return self.child.shape
+
+    # @property
+    # def shape(self) -> Union[TypeTuple[Any, ...], List[Any]]:
```

Comment:
Whats going on here?

Reply:
I'd like to leave this here - we're not sure where we're going to put it yet but we need to have both documentation and implementation for how we handle shape information on pointers.